tests/ignite/handlers/test_ema_handler.py

```diff
@@ -238,14 +252,14 @@ def _test_ema_final_weight(model, device, ddp=False, interval=1):
 
 
 @pytest.mark.parametrize("interval", [1, 2])
-def test_ema_final_weight_cpu(get_dummy_model, interval):
+def test_cpu(get_dummy_model, interval):
```

Comment:
```suggestion
def test_ema_final_weight_cpu(get_dummy_model, interval):
```
I think the name was OK.